ld: error: undefined symbol: _Unwind_Resume
>>> referenced by osinfo-db-util.c
>>>               tools/f9d35d4@@osinfo-db-validate@exe/osinfo-db-util.c.o:(osinfo_db_get_system_path)

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -202,7 +202,6 @@ osinfo_db_tools_check_cflags = [
   '-Wframe-larger-than=4096',
   '-Wno-overlength-strings',
   '--param=ssp-buffer-size=4',
-  '-fexceptions',
   '-fasynchronous-unwind-tables',
   '-fdiagnostics-show-option',
   '-funit-at-a-time',
